server/resources/[VORP]/[homestead_essentials]/homestead_jobs/shared/jobs.json

```diff
@@ -0,0 +1,133 @@
+{
+    "Jobs": {
```

Comment:
we should consider removing this parent key so we can have direct look ups from our keys O(1) @BuffsOverNexus thoughts?